source/QtCoralNetToolboxWidget.py

```diff
@@ -853,13 +857,29 @@ def taglabImport(self):
 
         """
         try:
-            # Import the points
-            self.parent().activeviewer.annotations.importCoralNetCSVAnn(self.predictions_file,
-                                                                        self.parent().project,
-                                                                        self.parent().activeviewer.image)
+            # Import the points, with project labels and active_image
+            labels = self.parent().project.labels
+            active_image = self.parent().activeviewer.image
+
+            imported_points = self.parent().activeviewer.annotations.importCoralNetCSVAnn(self.predictions_file,
+                                                                                          labels,
+                                                                                          active_image)
+
+            self.parent().labels_widget.setLabels(self.parent().project, self.parent().activeviewer.image)
+            self.parent().groupbox_blobpanel.blockSignals(True)
+
+            for point in imported_points:
+                self.parent().project.addPoint(active_image, point, notify=True)
+
+            self.parent().groupbox_blobpanel.blockSignals(False)
+            self.parent().activeviewer.drawAllPointsAnn()
+
```

Comment:
@gaiapavoni @massimomartinelli

Unsure about getting the point's AI confidence scores to show up in the Panel correctly...

Reply:
Gaia worked on this matter, but if I remember well the confidence scores are loaded correctly using a CoralNet file. We will check and fix this ASAP. Thank you for the PR and to point out this issue.